en/docs/Learn/APIController/ci-cd-with-wso2-api-management.md

```diff
@@ -20,13 +20,34 @@ different ways of addressing the problem. The above diagram depicts a generic so
 -   API Publishers: Publish APIs to users
 -   DevOps: Control the deployment process
 
-API Developers and Publishers work with a version control system, which acts as a single source of truth for the pipeline. 
+API Developers and Publishers work with a version control system, which acts as a single source of truth for the pipeline.   
+
+## API Publisher based CI/CD
+
 API Developers can use WSO2 API Manager's Publisher to create APIs. CI/CD for API Manager relies on a Version Control system that acts as a Single Source of Truth for the pipeline. After APIs are exported from one environment, promoting it to the other environment is done via the `apictl`. It is capable of handling 
-environment-related configurations and can promote the API seamlessly to other environments via a single command.  
+environment-related configurations and can promote the API seamlessly to other environments via a single command.
```

Comment:
Shall we change this as,

API Developers can use WSO2 API Manager's Publisher Portal to create an API in a lower environment. CI/CD for API Manager relies on a Version Control system that acts as a Single Source of Truth for the pipeline. So, after the API is exported from the lower environment, the exported API artifacts can be committed to a source code management repository and the promotion of the APIs to the upper environments can be done after running the tests in the lower environment. This process of promoting the API seamlessly to multiple environments can be automated via the `apictl` and well-configured CI/CD tools. `apictl` makes this more simpler by handling per environment-related configurations.

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/677/commits/eb59e1a0122cdb90be9152a172194ca6a78ca83c